lambda/otel/layer/Gemfile

```diff
@@ -1,9 +1,4 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-
-## keep in case we need custom packages again
-# source 'https://rubygems.pkg.github.com/solarwinds' do
-# end
```

Comment:
seems we should be keeping this in case of custom packages?

Reply:
Current approach probably won't work with this custom package definition. I can updated it once we need to revisit it.